src/posix/platform/radio_spinel.hpp

```diff
@@ -513,11 +514,12 @@ class RadioSpinel : public HdlcInterface::Callbacks
     uint32_t GetRadioChannelMask(bool aPreferred);
 
     /**
-     *  This method processes a received Spinel frame.
+     * This method processes a received Spinel frame.
+     *
+     * The newly received frame is available in `RxFrameBuffer` from `SpinelInterface::GetRxFrameBuffer()`.
      *
-     * @param[in] aFrameBuffer The frame buffer constaining the newly received frame.
      */
-    void HandleSpinelFrame(HdlcInterface::RxFrameBuffer &aFrameBuffer);
+    void HandleReceivedFrame(void);
```

Comment:
Prefer to pass arguments from function parameters rather than implicitly passing via members.

Reply:
Are there any benefits to pass the arguments through function parameters? `SpinelInterface` already provide a function `GetRxFrameBuffer()` to up layer to get the received frames. So up layer can use this unified API to get the received frames.